src/libs/vtkh/vtkm_filters/vtkmProbe.cpp

```diff
@@ -0,0 +1,55 @@
+#include "vtkmProbe.hpp"
+#include <vtkm/filter/resampling/Probe.h>
+#include <vtkm/cont/DataSetBuilderUniform.h>
+
+namespace vtkh
+{
+
+void
+vtkmProbe::dims(const Vec3f dims)
+{
+  m_dims = dims;
+}
+
+void
+vtkmProbe::origin(const Vec3f origin)
+{
+  m_origin = origin;
+}
+
+void
+vtkmProbe::spacing(const Vec3f spacing)
+{
+  m_spacing = spacing;
+}
+
+void
+vtkmProbe::invalidValue(const vtkm::Float64 invalid_value)
+{
+  m_invalid_value = invalid_value;
+}
+
+vtkm::cont::DataSet
+vtkmProbe::Run(vtkm::cont::DataSet &input)
+{
+  vtkm::filter::resampling::Probe probe;
+
+  std::string name = "coords";
+  vtkm::Id3 dims = 3;
+  if(m_dims[2] == 0)
+    dims = 2;
+
+  //move origin to lower left corner
+  m_origin[0] = m_origin[0] - (m_dims[0]/2);
+  m_origin[1] = m_origin[1] - (m_dims[1]/2);
+  m_origin[2] = m_origin[2] - (m_dims[2]/2);
```

Comment:
I want to make sure I understand the origin shift. 
Does VTK-m create Uniform Datasets from the center of the Volume?

Reply:
In general, Conduit also starts with an origin + dx,dy,dz. Which is lower left corner with respect to an overall coordinated system. 

That said the blueprint example you are using may be centering things at 0 and have volume that covers both negative and positive parts of the overall coordinate system.

Want to make sure we aren't confusing a specific case for testing with what we need for the general case.